tests/test-cmd/test.run.js

```diff
@@ -281,6 +281,7 @@ describe('run', () => {
       const args = {
         addonId: 'some-addon@test-suite',
         client,
+        // $FLOW_IGNORE: fake a Firefox ChildProcess using an EventEmitter for testing reasons.
```

Comment:
after discussing on IRC, we think an interface would be too verbose to cover this case